.github/workflows/build.yml

```diff
@@ -12,15 +12,16 @@ jobs:
               with:
                   node-version: 20
             - name: Checkout Repository
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
```

Comment:
out of curiosity, why was it needed?

Reply:
for funzies, no specific reason, just to stay with the latest version